compute/cloud-client/src/main/java/compute/disks/CloneEncryptedDisk.java

```diff
@@ -0,0 +1,108 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compute.disks;
+
+// [START compute_disk_clone_encrypted_disk]
+
+import com.google.cloud.compute.v1.CustomerEncryptionKey;
+import com.google.cloud.compute.v1.Disk;
+import com.google.cloud.compute.v1.DisksClient;
+import com.google.cloud.compute.v1.InsertDiskRequest;
+import com.google.cloud.compute.v1.Operation;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CloneEncryptedDisk {
+
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Project ID or project number of the Cloud project you want to use.
+    String project = "YOUR_PROJECT_ID";
```

Comment:
```suggestion
    String projectIdOrNumber = "YOUR_PROJECT_ID_OR_NUMBER"
```

Reply:
We have followed this variable naming and comment practice across all compute samples. Modifying it right now woudn't be compatible with the other samples.
I would prefer to leave it as such! 
wdyt?

Reply:
Ok, what I proposed will be a larger change than I thought. Will defer to what's existing.